### ISSUE TITLE
Other Error: name 'path' is not defined

### ISSUE THREAD

**Issue by wergio** [kuchel77/diskspace#3]
I get this error (HASSIO in docker in Rpi4)
```
WARNING (SyncWorker_14) [custom_components.diskspace.sensor] Other Error: name 'path' is not defined
```
my conf:
```
sensor:
  - platform: diskspace
    name: SD Card
    path: "/"
    unit_of_measure: "GB"
```

**Comment by kuchel77** [comment]
Hi, 

I thought it should be fixed in 0.6 but it didn't seem to be. It should be fixed in 0.7.

If you check whether or not it has the line

```
_LOGGER.debug("Path %s", self._path)
```

rather than

```
_LOGGER.debug("Path %s", path)
```

Then the error should disappear.

**Comment by wergio** [comment]
fixed with ver 0.7 thanks
